server/resources/server.html

```diff
@@ -16,29 +22,35 @@
 		<label>{{.PlayingData.Station}} </label> <br>
 		<label>  {{.PlayingData.Title}} </label> <br>
 
-		<form action="/radio/play" method="post">
+		<form action="/radio/api/play" method="post">
 			<input class="mediaControlBtn" type="submit" name="playRadio" value="Play">
 		</form>
-		<form action="/radio/stop" method="post">
+		<form action="/radio/api/stop" method="post">
 			<input class="mediaControlBtn" type="submit" name="stopRadio" value="Stop">
 		</form>
+		<form action="/radio/api/next" method="post">
+			<input class="mediaControlBtn" type="submit" name="nextRadio" value="Next">
+		</form>
+		<form action="/radio/api/prev" method="post">
+			<input class="mediaControlBtn" type="submit" name="prevRadio" value="Prev">
+		</form>
 
-		<form action="/radio/set" method="post">
+		<form action="/radio/api/set" method="post">
```

Comment:
I wonder if it makes sense at all to hardcode list of stations in the HTML at all. I'd decouple front end from the backend completely. I'd rather request the radio station via REST API in JSON format? Same goes for other data/actions (like play/stop/next/prev). The way it is built now it refreshes the page every time you do any action and that's not really best way to do modern apps.